apps/macos/react-native.config.js

```diff
@@ -18,5 +18,10 @@
 const path = require('path');
 
 module.exports = {
-  reactNativePath: path.dirname(require.resolve('react-native-macos/package.json')),
+  project: {
+    ios: {
+      project: 'src/FluentUITester.xcworkspace'
+    }
+  },
```

Comment:
Why is this iOS over here?

Reply:
It's a bug in `@react-native-community/cli`. I've opened an issue in https://github.com/react-native-community/cli/issues/1054.

Reply:
So we still need this for the macOS test app for autolinking, even though we don't have an iOS project here?

Reply:
Yes, I haven't looked much into it but I believe macOS autolinking reuses much of the iOS code.